Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM alpine:3.13 as otelcol
+FROM alpine:3.14.0 as otelcol
```

Comment:
Reading the [Alpine 3.14 release notes](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0), I'm a bit worried Docker images built from Alpine 3.14 will not work on most current platforms. Probably worth first building locally and checking, it's quite possible I'm misunderstanding the below.

> Therefore, as of writing, the following platforms are not suitable as Docker hosts for Alpine Linux 3.14.0, due to containing out-of-date libseccomp: Alpine Linux 3.12 and earlier, Amazon Linux 1 or 2, CentOS 7 or 8, Debian 10 or earlier without backports enabled, Raspbian stable or earlier, openSUSE Leap (any release), Ubuntu (any release), and Windows.

Reply:
Ok, valid point let's just park this update aside. No need to rush for it.